Monika After Story/game/script-topics.rpy

```diff
@@ -3641,13 +3641,14 @@ label monika_trolley:
     m 1eua "Oh, cool. I love thinking about these sorts of thought experiments."
     m 1euc "I guess we're talking about real people, right? I wouldn't have a particular preference if I knew they didn't have free will."
     m 1dsc "Hmmm..."
-    m 1eua "If we were talking about the classic problem, of course I would switch."
-    m 1rkc "But if we're talking about real people..."
-    m 3esc "I wouldn't switch."
-    m 4esc "That's the only logical choice, if you think about it."
-    m 1euc "It feels nice to say that you'd save the most people, doesn't it? But people only say that because nobody in the example matters to them."
-    m "If they actually cared about the person, they'd kill the other five in a heartbeat."
-    m 2tsb "Do you know anyone who would want a self-driving car to save other people by killing the driver? Didn't think so, [player]."
+    m 3esa "The classic trolley problem has us choose between letting the trolley run over 5 people, or switching to the track where only 1 person will be killed".
+    m 1esb "I think this problem is too easy. Of course I would switch."
+    m 1esc "It takes a rare kind of moral cowardice not to save 4 lives, just to avoid personal responsibility for that one person's death."
```

Comment:
If we're talking about the classic variant, then switching will save 5 lives.

Reply:
There's 5 people on one track and 1 on the other. Either 1 dies or 5 do. This leaves you with net value of 4.

Reply:
Imo it should be `4 more lives` or just `5 lives`. Using the net value makes this more confusing and makes it seem like a different problem.

Reply:
Alright, point taken. 5 it is.